src/NewTwitchApi/Webhooks/WebhooksSubscriptionApi.php

```diff
@@ -138,12 +141,16 @@ private function subscribe(string $topic, string $callback, string $bearer = nul
         ]);
     }
 
-    private function unsubscribe(string $topic, string $callback): void
+    private function unsubscribe(string $topic, string $callback, string $bearer = null): void
     {
         $headers = [
             'Client-ID' => $this->clientId,
         ];
 
+        if (!is_null($bearer)) {
```

Comment:
Since Authorization is required, this parameter should be passed without an if statement